src/assets/css/App.css

```diff
@@ -4,6 +4,8 @@ body{
 .App {
   text-align: left;
   border-top: 1px solid #D9DBDC;
+  position: relative;
+  min-height: 700px;
```

Comment:
This `min-height` can break on smaller devices :/ And for sure is not "generic" solution. But maybe we can live with it for now. I will test how it looks now.